app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, number):
+        if isinstance(number, Distance):
+            return Distance(self.km + number.km)
+        return Distance(self.km + number)
+
+    def __iadd__(self, number):
+        if isinstance(number, Distance):
+            self.km += number.km
+        else:
+            self.km += number
+        return self
+
+    def __mul__(self, number):
+        if isinstance(number, Distance):
```

Comment:
`number` can not be of type `Distance` in this method, because if you multiply km * km you get square km